docs/CustomApp.md

```diff
@@ -244,12 +248,14 @@ const App = () => (
 -export default App;
 +export default withContext(
 +   {
-+       authProvider: PropTypes.func,
++       authProvider: PropTypes.object,
 +   },
 +   () => ({ authProvider })
 +)(App);
 ```
 
 Note that this example still uses `<Resource>`, because this component lazily initializes the store for the resource data.
 
+Currently the `<Notification>` component is required (See [issue #5026](https://github.com/marmelab/react-admin/issues/5026)).
```

Comment:
The notification component will always be required for optimistic support so I think you can remove this line

Reply:
Done !
Have to say I find it strange that a component called `Notification` is required for optimistic update. If it can't be changed, maybe it should be renamed ?